scripts/download-sync-diff.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+# Copyright 2022 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+SHA1=$(curl http://fileserver.pingcap.net/download/refs/pingcap/tidb-tools/master/sha1)
+echo "will download tidb-tools at ${SHA1} to get sync_diff_inspector"
+curl -C - --retry 3 -o /tmp/tidb-tools.tar.gz http://fileserver.pingcap.net/download/builds/pingcap/tidb-tools/${SHA1}/centos7/tidb-tools.tar.gz
```

Comment:
Better to use an external address.

Reply:
DM integration test uses master build to shorten the response time when sync-diff fixes a bug. 

@CharlesCheung96 Can we let CI environment ro prepare the downloading?

Reply:
Sure, such binaries will be prepared in advance in the CI environmen. Besides, an external address can be used to allow developers to run integration tests locally.